.github/license_config.yml

```diff
@@ -0,0 +1,17 @@
+license:
+  main: apache-2.0
+  report_missing: true
+  category: Permissive
+copyright:
+  check: true
+exclude:
+  extensions:
+    - yml
```

Comment:
Do we really want to exclude `yml/yaml`?  Is it scancode or some other check that flags YAML files needed copyright & license headers?

Reply:
every file is scanned, if we add yaml files we will have to change 100s of them in the tree and add license/copyright.

Reply:
Aren't we doing that today?  I know I got a message about adding license/copyright to my .github/workflow/<FOO>.yml.

Even you have it in this PR:

```
.github/license_config.yml missing copyright.
.github/workflows/license_check.yml missing license.
.github/workflows/license_check.yml missing copyright.
```

Reply:
> Even you have it in this PR:

yeah, we exclude yaml, but not yml in ci-tools

Reply:
> yeah, we exclude yaml, but not yml in ci-tools, which means that something is wrong in the script, will take a look.

actually no, the above is from the ci-tools checker which does not exclude .yml, the action does things correctly.
In any case, I would not be changing policy now, first get this in, then the config can be changed based on needs.